pandas/io/formats/format.py

```diff
@@ -1198,6 +899,209 @@ def _get_column_name_list(self) -> List[str]:
         return names
 
 
+class DataFrameRenderer:
```

Comment:
can you add a doc-string here and explain that these to_* are being called in core/frame/to_* ultimately.

Reply:
Added